src/config/dynamic-theme-fixes.config

```diff
@@ -9494,6 +9494,13 @@ INVERT
 
 ================================
 
+radareu.cz
+
+INVERT
+#radarMap
```

Comment:
```suggestion
#radarMap .leaflet-map-pane
```

These div only contains the panels etc and shouldn't revert all the other UI elements.